thoughtis/cata-co-authors-plus
# Update for WordPress 6.6

### What Was Accomplished

- Updated dependencies to use the version tagged `wp-6.6` and rebuild all assets.

### How To Test
- Ensure when editing a guest author with the block editor, their profile URL is available in the right side panel and the URL is correct.
- When editing a guest author you can add the Co-Author Tagline or Social Links blocks and they will correctly populated in the rendered content.
